src/MagnumPlugins/AssimpImporter/Test/AssimpImporterTest.cpp

```diff
@@ -1426,47 +1426,10 @@ void AssimpImporterTest::skinMergeEmpty() {
 }
 
 void AssimpImporterTest::skinMerge() {
-    if(!supportsAnimation(".fbx"))
-        CORRADE_SKIP("FBX skinning is not supported with the current version of Assimp");
-
-    Containers::Pointer<AbstractImporter> _importer = _manager.instantiate("AssimpImporter");
-    /* Disabled by default */
-    CORRADE_VERIFY(!_importer->configuration().value<bool>("mergeSkins"));
-    CORRADE_VERIFY(_importer->openFile(Utility::Directory::join(ASSIMPIMPORTER_TEST_DIR, "skin.fbx")));
-
-    /* Manually construct bone data and add it to a scene */
-    const aiScene* original = static_cast<const aiScene*>(_importer->importerState());
-    CORRADE_VERIFY(original);
-    aiScene* scene{};
-
-    /* aiCopyScene() is for some reason not present in builds without export
-       functionality (which we have on Windows CIs). The code inside does this
-       and then sets some private mIsCopy field which we don't care about.
-       Unfortunately, <assimp/SceneCombiner.h> is available only since 4.0, so
-       on the ancient 3.2 use the C API (and assume nobody needs the insane corner case of running the test against Assimp 3.2 that has export APIs
-       disabled). */
-    #ifdef ASSIMP_HAS_SCENE_COMBINER
```

Comment:
The `#include` block related to this should get cleaned up as well. I'll do it during merge.

Otherwise looks great, thanks a lot! :+1: